src/reference/asciidoc/kafka.adoc

```diff
@@ -1610,9 +1610,42 @@ Starting with _version 2.1_, type information can be conveyed in record `Headers
 In addition, the serializer/deserializer can be configured using Kafka properties.
 
 - `JsonSerializer.ADD_TYPE_INFO_HEADERS` (default `true`); set to `false` to disable this feature on the `JsonSerializer` (sets the `addTypeInfo` property).
+- `JsonSerializer.TYPE_MAPPINGS` (default `empty`); see below
```

Comment:
Maybe we should add period in the end of sentence?
At least that is what I see in other bullets here...